extra/apm_docs_landing.html

```diff
@@ -0,0 +1,34 @@
+<style>
+#content > .ulist .itemizedlist {
+  visibility: hidden;
+}
+</style>
+
+<div class="w-100 mb-5">
+  <h2>APM user guide</h2>
+  <p>
+    Learn how to use APM in the <a href="/guide/en/observability/master/apm.html">Observability guide</a>.
+  </p>
+</div>
+<div class="w-100 mb-5">
+  <h2>APM agents</h2>
+  <ul>
+    <li><a href="/guide/en/apm/agent/android/current/intro.html">Android Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/go/current/introduction.html">Go Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/swift/current/intro.html">iOS Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/java/current/intro.html">Java Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/dotnet/current/intro.html">.NET Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/nodejs/current/intro.html">Node.js Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/php/current/intro.html">PHP Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/python/current/getting-started.html">Python Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/ruby/current/introduction.html">Ruby Agent Reference</a></li>
+    <li><a href="/guide/en/apm/agent/rum-js/current/intro.html">Real User Monitoring JavaScript Agent Reference</a></li>
+  </ul>
+</div>
+<div class="w-100 mb-5">
+  <h2>APM extensions</h2>
+  <ul>
+    <li><a href="/guide/en/apm/lambda/current/aws-lambda-arch.html">Monitoring AWS Lambda Functions</a></li>
```

Comment:
It's not the name of the book, but the name of the extension is "APM AWS Lambda extension." You okay with using that?

```suggestion
    <li><a href="/guide/en/apm/lambda/current/aws-lambda-arch.html">APM AWS Lambda extension</a></li>
```